backend/capellacollab/sessions/files/routes.py

```diff
@@ -33,7 +33,7 @@ def list_files(
     except Exception:
         log.exception("Loading of files for session %s failed", session.id)
         raise fastapi.HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            status_code=status.HTTP_404_NOT_FOUND,
```

Comment:
Here, I'd expect another error code. 404 would mean that the file doesn't exist, but this is not a statement that we can make. We just don't know if the file exists. Here, I'd actually say that internal server error is the correct response.

Reply:
Fair point. I think I would go for 422 then, since it may be a user error and is an error outside of the backend itself.